sys/oneway-malloc/Kconfig

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2021 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+config MODULE_ONEWAY_MALLOC
+    bool "One way malloc"
+    depends on TEST_KCONFIG
+    help
+        A malloc implementation without free for boards where the
+        toolchain does not implement dynamic memory allocation.
```

Comment:
Missing newline

Reply:
?

Reply:
Sorry, missed the fixup. I commented only looking at 4839f774a5e396a5e9bede43ffeaee31d81bf51b